frontend/views/containers/modals/SignUp.vue

```diff
@@ -161,8 +164,7 @@ export default {
         } catch (ex) {
           console.error('SignUp.vue submit() error:', ex)
           sbp('state/vuex/dispatch', 'logout')
-          this.form.response = ex.toString()
-          this.form.error = true
```

Comment:
If `form.error` isn't used anymore then it should also be removed from the `data ()` method

Reply:
removed